### ISSUE TITLE
OBSOLETE FEATURE: indexed objects for pgsphere

### ISSUE THREAD

**Issue by kimakan** [django-daiquiri/daiquiri#238]
I guess that at some point there were plans to use the extra column of type spoint(ra, dec) which is indexed and then, it can be used instead of ra, dec. I'm not quiet sure whether the implementation was completed. The code in question is 
https://github.com/django-daiquiri/daiquiri/blob/5ae2a14f68bfab0b0d6c261ebce3e6359d6594de/daiquiri/query/process.py#L178-L185

https://github.com/django-daiquiri/daiquiri/blob/5ae2a14f68bfab0b0d6c261ebce3e6359d6594de/daiquiri/query/utils.py#L132-L148

IMHO, the feature is obsolete because the indexes can be created directly for the `spoint()` expression. Meaning that instead of creating a new column `pos` as `spoint(ra, dec)`, creating an index for it and then trying in some hacky way to replace spoint(ra,dec) with the indexed column, we just create an index as 
```sql
Create index schema_table_spoint_ra_dec on schema.table  using gist (spoint(RADIANS(ra), RADIANS(dec)));
```
No need to create an extra column in the database and no need to connect ra/dec with the spoint(ra,dec) in the translated query.

Removing this feature would simplify daiquiri and handling of the database.

**Comment by jochenklar** [comment]
Maybe we just did not know enough about databases back then. Did you test if your solution is as fast as the one with the extra column?

**Comment by kimakan** [comment]
Yes, we have already tested (and still using) the index `spoint(RADIANS(ra), RADIANS(dec))` for gaia DR3. In the case of the cone search, the new solution is even faster (~3x).

**Comment by kimakan** [comment]
Solved with #329
